### PR TITLE
[TA] Apply v5.1.1 bug fix to 5.2.0-preview.1 

### PR DESCRIPTION
**Pull request by mssfang**
Cherry-pick the commit of 5.1.1 and apply the changes of v5.1.1 to v5.2.0-preview.1 

The merged commit PR: https://github.com/Azure/azure-sdk-for-java/pull/23410

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java [hunk 1]
```diff
@@ -211,6 +211,7 @@ private JobManifestTasks getJobManifestTasks(TextAnalyticsActions actions) {
                         entitiesTask.setParameters(
                             new EntitiesTaskParameters()
                                 .setModelVersion(action.getModelVersion())
+                                .setLoggingOptOut(action.isServiceLogsDisabled())
```
Comment (samvaity) [inline sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java:214]: Does the function name end with `opt - out`?

Comment (samvaity) [reply sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java:214]: nvm, just saw the referenced PR.

**Review by deyaaeldeen — APPROVED** [review]

File: sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,12 @@
 - We are now targeting the service's v3.2-preview.1 API as the default instead of v3.1.
 - Added support for Extractive Summarization actions through the `ExtractSummaryAction` type.
 
+## 5.1.1 (2021-08-10)
+### Bugs Fixed
+- Fixed the bug to support the default value `disableServiceLogs = true`, in the option bags, `AnalyzeHealthcareEntitiesOptions`, 
+  `RecognizePiiEntitiesOptions` and `RecognizePiiEntitiesAction`, 
+- Using UTF-16 code unit as the default encoding in the `Sentimant Analysis` and `Linked Entities Recognition` actions.
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:11]:
```suggestion
- Using UTF-16 code unit as the default encoding in the `Sentiment Analysis` and `Linked Entities Recognition` actions. 
```

File: sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeHealthcareEntitiesOptions.java [hunk 3]
```diff
@@ -39,7 +41,19 @@ public AnalyzeHealthcareEntitiesOptions setIncludeStatistics(boolean includeStat
     }
 
     /**
-     * Sets the value of {@code disableServiceLogs}.
+     * Gets the value of service logs disable status. The default value of this property is 'true'. This means,
+     * Text Analytics service won't log your input text. Setting this property to 'false', enables logging your input
+     * text for 48 hours, solely to allow for troubleshooting issues.
+     *
+     * @return true if logging input text service are disabled.
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeHealthcareEntitiesOptions.java:48]: maybe "true if service logging of input text is disabled"

**Review by kristapratico — APPROVED** [review]

**Pull request merged**